src/util/log.cc

```diff
@@ -542,28 +542,20 @@ std::ostream& operator<<(std::ostream& out, const std::exception_ptr& eptr) {
         // Print more information on some familiar exception types
         try {
             throw;
-        } catch (const seastar::nested_exception& ne) {
-            out << fmt::format(": {} (while cleaning up after {})", ne.inner, ne.outer);
-        } catch(const std::system_error &e) {
+        } catch (const std::system_error &e) {
             out << " (error " << e.code() << ", " << e.what() << ")";
-        } catch(const std::exception& e) {
+        } catch (const std::exception &e) {
```

Comment:
@avikivity 's coding style is type& x, not type &x. Please don't go against it in your patch.

Reply:
Agrrr... It's Clion's auto-formatting. I'll fix, NP.